python/basic-app/README.md

```diff
@@ -82,3 +82,21 @@ to
 ```
 from tempfile import TemporaryFile
 ```
+
+## Optional: Enable Google Chat app request verification
```

Comment:
Questions:

* Why show both methods? the newer APP_URL approach is easier and supported by the client libraries, maybe be more opinionated?
* Part of the reason for the new method is it is supposed to work with GCP's IAM policies. Given that the sample is framed as running on app engine, should that be the preferred approach?